pymodbus/client/serial_asyncio/README.md

```diff
@@ -1,4 +1,3 @@
 Patch for serial_asyncio, also submitted to the repo:
 
 https://github.com/pyserial/pyserial-asyncio/pull/94
-
```

Comment:
no

Reply:
We do NOT change in a copied module. Apart from that I believe it has nothing to do with the PR.

Reply:
Reverted.

Aside: the right way to do this is make a `git submodule` where one filters for the files needed

Reply:
Not its not, the problem is that upstream have become inactive, I submitted a couple of pull requests long time ago with a patch that solved an urgent problem.

Since there was no response we had to integrate it into our source with the patch. Git submodule is useful only if you refer to an existing repo.

Reply:
Okay gotchu.  It's a shame pySerial seems to be abandoned